Jasmine/Grid/ViewModels/GridViewModel.swift

```diff
@@ -2,32 +2,43 @@ import Foundation
 
 class GridViewModel: GridViewModelProtocol {
     /// Stores the grid data that will be used to display in the view controller.
-    private(set) var gridData: [Coordinate : String] = [:]
+    private(set) var gridData: [Coordinate: String] = [:]
```

Comment:
why don't implement `updateGridData` as a `didSet` for this property? So you don't have to manually call it each time you update the grid data.